lighthouserc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  ci: {
+    collect: {
+      url: ['http://localhost:3000/'],
+      startServerCommand: 'yarn start'
+    },
+    // leaving assertions out until other problems are fixed
+    // assert: {
+    //   preset: 'lighthouse:recommended'
+    // },
+    upload: {
+      target: 'temporary-public-storage'
```

Comment:
What's "temporary-public-storage"?

Reply:
It is so that the report is automatically uploaded to a public url on google cloud and is deleted in 7 days. It's so that below when you click details to the right of lighthouse it pulls it up and anyone with the link can view it.

Reply:
So all PR's must be reviewed within 7 days?

Reply:
Yeah on the default setting, but there is probably a way to change it if that is a problem.

Reply:
We'll change it if we run into a problem. could always to add a comment commit to refresh the PR.